tests/ut_featuremodel/featuremodeltest.cpp

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file featuremodeltest.cpp
+ * @copyright 2015 Jolla Ltd.
+ * @author Joona Petrell <joona.petrell@jolla.com>
+ * @date 2015
+ */
+
+#include "featuremodeltest.h"
+
+#include <QtTest/QtTest>
+#include <QtTest/QSignalSpy>
+#include "libssu/ssufeaturemodel.h"
+
+void FeatureModelTest::initTestCase(){
+
+}
+
+void FeatureModelTest::cleanupTestCase(){
+
+}
+
+void FeatureModelTest::testFeatures(){
+    SsuFeatureModel featureModel(0, ":/testdata");
+
+    // Features with no name or version were skipped (feature-noname.ini, feature-noversion.ini)
+    QCOMPARE(featureModel.count(), 2);
```

Comment:
I wonder if we should skip only entries without name, but when version is missing just show name in the lists?